app/scripts/Navy.js

```diff
@@ -7,7 +7,7 @@ const Navy = {
   },
 
   resizeDrawerHeight (target) {
-
+    if(!target) return;
```

Comment:
Caused errors in console, if resize was triggered, without target fully rendered